R/pkg/R/functions.R

```diff
@@ -259,6 +259,20 @@ setMethod("column",
           function(x) {
             col(x)
           })
+#' corr
+#'
+#' Computes the Pearson Correlation Coefficient for two Columns.
+#'
+#' @rdname corr
+#' @name corr
+#' @family math_funcs
+#' @export
+#' @examples \dontrun{corr(df$c, df$d)}
+setMethod("corr", signature(x = "Column", col1 = "Column", col2 = "missing", method = "missing"),
```

Comment:
Maybe we can add cor() as alias for corr(), as you did in https://github.com/apache/spark/pull/9489